bower.json

```diff
@@ -63,7 +63,8 @@
     "patternfly-bootstrap-treeview": "2.1.1",
     "angular": "1.5.11",
     "kubernetes-object-describer": "1.0.4",
-    "angular-sanitize": "1.5.11"
+    "angular-sanitize": "1.5.11",
+    "origin-web-common": "0.0.11"
```

Comment:
@jwforres we should be able to remove this resolution when we bump origin-web-catalog